doc/user/data/materialize_operator_chart_parameter.yml

```diff
@@ -0,0 +1,812 @@
+
+parameters:
+  - parameter: clusterd.nodeSelector
+    description: |
+      <red>Replace with description content here.</red>
```

Comment:
Will need devs to:
1. Specify which of these parameters should be user-facing.
2. For those that are user facing, should update the descriptions.